application-licensing-licensor/application-licensing-licensor-ui/src/main/resources/Licenses/Code/UIExtensionLicenseChecking.xml

```diff
@@ -0,0 +1,302 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<xwikidoc version="1.3" reference="Licenses.Code.UIExtensionLicenseChecking" locale="">
```

Comment:
By looking at https://github.com/xwikisas/application-licensing/tree/master/application-licensing-licensor/application-licensing-licensor-ui/src/main/resources/Licenses/Code I think the naming convention used so far is to have the UIX suffix for UIExtension pages. So in order to be consistent I would go for ``LicenseCheckingUIX``. **But**, I see that we already have a page ``MissingLicensesUIX`` which uses the same extension point (after header) so maybe we should extend that page instead of creating another one. WDYT?

Reply:
Since both use the same EP, probably is a good idea to merge the code in `MissingLicensesUIX`.